scanpy/external/pl.py

```diff
@@ -332,15 +332,14 @@ def scrublet_score_distribution(
     figsize: Optional[Tuple[float, float]] = (8, 3),
 ):
     """\
-    Plot histogram of doublet scores for observed transcriptomes and simulated doublets. 
+    Plot histogram of doublet scores for observed transcriptomes and simulated doublets.
+
+    The histogram for simulated doublets is useful for determining the correct doublet score threshold.
```

Comment:
```suggestion
    The histogram for simulated doublets is useful for determining the correct doublet
    score threshold. 
```

Bit long, seemed fine before